_unused_topics/cluster-logging-configuring-node-selector.adoc

```diff
@@ -45,20 +45,13 @@ spec:
       nodeSelector:  <2>
         logging: kibana
       replicas: 1
-  curation:
-    type: "curator"
-    curator:
-      nodeSelector:  <3>
-        logging: curator
-      schedule: "*/10 * * * *"
   collection:
     logs:
       type: "fluentd"
       fluentd:
-        nodeSelector:  <4>
+        nodeSelector:  <3>
         logging: fluentd
 ----
 <1> Node selector for Elasticsearch.
 <2> Node selector for Kibana.
-<3> Node selector for Curator.
-<4> Node selector for Fluentd.
```

Comment:
@kabirbhartiRH FYI - Removing curator-related lines from the example ClusterLogging CRD file and removing/renumbering the callouts. Many repetitions of this in the following files.